src/components/ItemList.styles.js

```diff
@@ -0,0 +1,53 @@
+import { StyleSheet } from 'react-native';
+import colors from '../assets/colors';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
```

Comment:
Isn't `flexDirection: row` the default value?

Reply:
Default is column in React native

Reply:
Oh in web it's row. I guess it make sense though.